frontend/web/src/pages/adminproperty/AdminProperty.js

```diff
@@ -193,53 +333,205 @@ export default function AdminProperty() {
 
     setSubmitting(true);
     try {
+      try {
+        const payload = builder.build ? builder.build() : builder;
+        const sizeBytes = new Blob([JSON.stringify(payload)]).size;
+        console.log(
+          "Payload size (bytes):",
+          sizeBytes,
+          "≈",
+          (sizeBytes / 1024 / 1024).toFixed(2),
+          "MB"
+        );
+      } catch {}
+
       await submitAccommodation(navigate, builder);
     } finally {
       setSubmitting(false);
     }
   };
 
+  const hasError = (fieldName) => touched[fieldName] && errors[fieldName];
+
   return (
     <form className="adminproperty-form" onSubmit={onSubmit}>
       <div className="adminproperty-group">
-        <label>What kind of space do your guests have access to?</label>
+        <label>What kind of space do your guests have access to</label>
```

Comment:
ik zag dat je de vraagteken had weggehaald maar in my opinion was hij wel nice om te hebben

```suggestion
        <label>What kind of space do your guests have access to?</label>
```